From: Romain Perier <mrpouet@gentoo.org>
Date: Sun, 8 Nov 2009 19:27:56 +0100
Subject: Fix conflicting types for games_sound_init

The problem was caused by two differents functions, matching the same name, 
but without the same proto.
Usually the commonly way is to prefix static functions by a '_' because this is useful only
for the implementation.

---
 libgames-support/games-sound.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

--- a/libgames-support/games-sound.c
+++ b/libgames-support/games-sound.c
@@ -157,7 +157,7 @@ games_sound_canberra_play (const char *sound_name,
 
 /* Initializes the games-sound support */
 static void
-games_sound_init (void)
+_games_sound_init (void)
 {
   const int audio_rate = MIX_DEFAULT_FREQUENCY;
   const int audio_format = MIX_DEFAULT_FORMAT;
@@ -272,7 +272,7 @@ games_sound_play_for_screen (const gchar * sound_name,
   if (!sound_enabled)
     return;
   if (!sound_init)
-    games_sound_init ();
+    _games_sound_init ();
 
   games_sound_sdl_play (sound_name);
 #endif
